### DISCUSSION TITLE
Mesa Dev Session Notes (February 2022)

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1173] [General]
[Matrix discussion established](https://matrix.to/#/!CAAdpmuJhdNJkxurrp:matrix.org?via=matrix.org) and the link was added to the README (thanks @EwoutH and @rht)

### Milestones for Next Dev Session (19 March) 

@tpike3 
- #1148 
- #1171 related to #1130

@jackiekazil 
- #1076

@EwoutH 
- #1170
- #1172 (complete)

@rht
Ensure compliance with standards and best practices

 
### Milestones on or before next release Quartzsite 1.0 (18 June) 

**agent activation by type @rht, @EwoutH** 
-  #1162 
-  #1142 

**improve CI**
 
- #1165 
- #1166 
- #1163 
- #1108 

### Larger initiatives: 
- Google Summer of Code application submitted
- - #1167 
- There is lots of curiosity about @jackiekazil's research on feature comparison across platforms (Mesa, MASON, Netlogo etc) and work on AWS extension
- SFI is still working on their portion to se can get a Mesa tutorial on Complexity Explorer.

**Comment by aiwhoo** [comment]
@tpike3, looking forward to the mesa tutorial.

Our team has added features and would like to integrate. Some of which include:

In the continuous space, adding 
1) a customizable background image for the canvas' grid. (simple_continous_canvas.js)
2) multiple agent shapes --- (smile face, arrow, default, star, pentagon, plane, ant, fish) (simple_continous_canvas.js)
3) additional agent functions common in agent-based language (face(agent), facexy(x,y), forward(speed), backward(speed), setxy(x,y), distancexy. (agent.py)

Our goal is to remove the need for a simple_continous_canvas.js, so that everything can be controlled within python.

**Reply by jackiekazil** [reply]
what prs?

**Reply by aiwhoo** [reply]
Haven't gotten there yet! Sorry, we're really new to open source.

**Reply by jackiekazil** [reply]
no worries at all. I wanted to make sure there wasn't something i was missing.

**Reply by tpike3** [reply]
@aiwhoo  That is very cool..... I have to build the discussion announcing it but could you come to our next dev session to discuss. 

It will be 19 March at 0730 via TBD

Our strategic goal is the best possible ABM ecosystem. I contend no one really knows what that looks like which will require a lot of intriguing discussions. We also put a new front end as #1 on our google simmer of code idea list #1167

**Comment by jackiekazil** [comment]
Do we have some thing officially scheduled on March 19? (Wondering if there is something we can put it up to add to our calendar.)

**Comment by tpike3** [comment]
I will post it today
